### PR TITLE
Fix and unify example docs

### PR DESCRIPTION
**Pull request by skonto**
### What changes were proposed in this pull request?
Add a GKE deployment example for the apps where that is missing and unify structure for several apps.

### Why are the changes needed?
This part was marked as TODO. Users also faced issues deploying the `sensor-data-scala` app.

### Does this PR introduce any user-facing change?
No.

### How was this patch tested?
Run the apps and documented the steps.

### CODE REVIEW

File: examples/sensor-data-scala/README.md [hunk 1]
```diff
@@ -1,81 +1,242 @@
...
+- Send the file to the HTTP ingress using `curl` (see deployment example later on).
 
-To send data to the HTTP ingress, do the following:
+To send data to the file ingress, use the following shell script found in the project root directory:
 
-- Get `sensor-data` ingress HTTP endpoint with `kubectl cloudflow status sensor-data-scala`
+    ./load-data-into-pvc.sh
 
-In the example output below the HTTP endpoint would be `docker-registry-default.my.kubernetes.cluster/sensor-data-http-ingress`:
+The shell script will load a number of files from the `test-data` directory and the ingress will continuously read those files and emit their content to the merge streamlet.
```
Comment (olofwalker) [inline examples/sensor-data-scala/README.md]:
```suggestion
The shell script will load several files from the `test-data` directory, and the ingress will continuously read those files and emit their content to the merge streamlet.
```

File: examples/taxi-ride/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ $ clean
 $ buildAndPublish
 ```
 
-The above will build the application and publish application Docker images to the Docker registry, as configured in `target-env.sbt`.
+The above will build the application and publish application Docker images to the Docker registry, as configured in `target-env.sbt`./c
```
Comment (olofwalker) [inline examples/taxi-ride/README.md]:
```suggestion
The above command builds the application and publish application Docker images to the Docker registry, as configured in `target-env.sbt`./c
```

File: examples/sensor-data-scala/README.md (hunk 1 above)
Comment (skonto) [reply examples/sensor-data-scala/README.md]: This pre-existed will fix it.

File: examples/taxi-ride/README.md (hunk 2 above)
Comment (skonto) [reply examples/taxi-ride/README.md]: This pre-existed will fix thanks.

**Pull request merged**